### DISCUSSION TITLE
`AppIntroBaseFragment` argument handling

### DISCUSSION THREAD

**Discussion by Fabi755** [AppIntro/AppIntro#949] [General]
While I creating PR #948 I have seen there is a lot of argument handling overhead.

The `SliderPage` has only native types (String, Int, CharSequence) and is already marked as `data class`.
I think it is easier to serialize or parcel the whole pager model and put them into a bundle to use them by creating **and** instance saving.

What do you think?

**Comment by cortinico** [comment]
>  I think it is easier to serialize or parcel the whole pager model and put them into a bundle to use them by creating **and** instance saving.

Indeed 👍 Feel free to refactor this 🙏
